libvast/vast/system/accountant.hpp

```diff
@@ -81,6 +70,20 @@ using accountant_type = caf::typed_actor<
   caf::reacts_to<telemetry_atom>>;
 // clang-format on
 
+/// @relates accountant
+struct accountant_state {
+  static constexpr const char* name = "accountant";
+  accountant_type::stateful_pointer<accountant_state> self;
```

Comment:
It seems like this pointer is never initialized. The state can have a constructor that takes the self pointer during construction.

Reply:
For some reason I thought this would work even without an explicitly defined constructor. Alas, it does not.